src/CanSatKitRadio.cpp

```diff
@@ -466,6 +466,10 @@ bool Radio::transmit(const char* str) {
   return transmit(str, strlen(str));
 }
 
+bool Radio::transmit(String str) {
+  return transmit(str.c_str(), strlen(str.c_str()));
```

Comment:
maybe it would be better to use `str.length()` method to avoid `strlen` pass of the string

Reply:
fair point, will do

Reply:
we've simplified API for the strings - now to send `const char*` length parameter is no longer there. After the rebase just remove second argument and we can merge that :+1:

Reply:
Yep, 1ff35bf did that I think